### PR TITLE
cross-mio files from Papa glider gp_537 and gp_575 recoveries

### PR DESCRIPTION
**Pull request by cdobs**
Papa glider gp_537 and gp_575 recoveries

### CODE REVIEW

File: vessel/shiplist.csv [hunk 1]
```diff
@@ -63,3 +63,4 @@ V,R/V,Vema,31VE,LDEO,J8RU,,7738383,,,,,(N),Y
 WJ,M/V,Warren Jr.,,BHC,,,,,,,,Y,N
 WOO,F/V,Wahoo,,Meredith,,,,,,,,Y,N
 W,R/V,Wecoma,32WC,OSU,WSD7079,303014000,7604439,56.4,7.2,6,,(N),Y
+ZP,R/V,Zephyr,,Zephyr Northwest,WDG9976,338142000,8992845,24,5.1,2.4,,Y,N
```
Comment (snwhite) [inline vessel/shiplist.csv:66]: Looks good.

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -271,6 +271,7 @@ TS20200301,F/V Tenacious II,2020-03-01T20:00:00,2020-03-02T15:00:00,Pioneer 13b
 WJ20200323,M/V Warren Jr.,2020-03-23T00:00:00,2020-03-27T23:59:00,Pioneer 13c AUV deployments
 AJ20200506,R/V Adolf Jensen,2020-05-06T12:00:00,2020-05-08T12:00:00,Irminger glider gi_525 recovery
 TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries
+ZP20200604,R/V Zephyr,2020-06-04T08:00:00,2020-06-08T23:59:59,Papa Glider gp_537 and gp_575 recoveries
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:274]: Looks good.

Comment (spetillo) [reply cruise/CruiseInformation.csv:274]: Looks good.

File: vessel/shiplist.csv (hunk 1 above)
Comment (spetillo) [reply vessel/shiplist.csv:66]: Also looks good.

**Pull request merged**